src/main/java/spoon/reflect/factory/CodeFactory.java

```diff
@@ -269,6 +270,19 @@ public CodeFactory(Factory factory) {
 		return literal;
 	}
 
+	/**
+	 * Creates a TextBlock with the given string value.
+	 * @param value
+	 * 		the string value of the literal
+	 * @return a new literal
+	 */
+	public CtTextBlock createTextBlock(String value) {
+		CtTextBlock textblock = factory.Core().createTextBlock();
+		textblock.setValue(value);
+		// TODO set textblock.setType
```

Comment:
fix or remove?

Reply:
Thank you very much for pointing out. I have made some changes. Kindly give them a look.